Including .NET sample for raw message subscriber (#4505)
dapr/docs: daprdocs/content/en/developing-applications/building-blocks/pubsub/pubsub-raw.md

To disable CloudEvent wrapping, set the `rawPayload` metadata to `true` as part of the publishing request. This allows subscribers to receive these messages without having to parse the CloudEvent schema.

> {{< tabs curl ".NET" "Python SDK" "PHP SDK">}}

Reviewer: ```suggestion
{{< tabs curl ".NET" "Python" "PHP">}}
```

{{% codetab %}}
```bash
curl -X "POST" http://localhost:3500/v1.0/publish/pubsub/TOPIC_A?metadata.rawPayload=true -H "Content-Type: application/json" -d '{"order-number": "345"}'
```
{{% /codetab %}}

{{% codetab %}}

```csharp
using Dapr.Client;

> using Shared;

Reviewer: What does this using refer to? It can probably be removed.

var builder = WebApplication.CreateBuilder(args);
builder.Services.AddControllers().AddDapr();

var app = builder.Build();

> app.MapGet("/", () => "Publisher API");

Reviewer: I think this GET method is not used in the sample, so this can be removed.

### Programmatically subscribe to raw events

> When subscribing programmatically, add the additional metadata entry for `rawPayload` - `isRawPayload` on .NET - so the Dapr sidecar automatically wraps the payloads into a CloudEvent that is compatible with current Dapr SDKs.

Reviewer: ```suggestion
When subscribing programmatically, add the additional metadata entry for `rawPayload` so the Dapr sidecar will automatically wrap the raw payload into a CloudEvent that is compatible with current Dapr SDKs. For .NET, this metadata entry is called `isRawPayload`.
```

Reviewer 2: We try to stay away from the future tense and use the present tense. 
` the Dapr sidecar automatically wraps the raw payload into a CloudEvent that is compatible with Dapr SDKs. For .NET, this metadata entry is called `isRawPayload`.`

Author: Resolved it by changing the text to: 

When subscribing programmatically, add the additional metadata entry for `rawPayload` to allow the subscriber to receive a message that is not wrapped by a CloudEvent. For .NET, this metadata entry is called `isRawPayload`.

> {{< tabs ".NET" "Python" "PHP SDK" >}}

Reviewer: ```suggestion
{{< tabs ".NET" "Python" "PHP" >}}
```

{{% codetab %}}

```csharp
using System.Text.Json;
using System.Text.Json.Serialization;

var builder = WebApplication.CreateBuilder(args);
var app = builder.Build();

> app.MapGet("/", () => "Subscriber API");

Reviewer: This MapGet can be removed

    default: /dsstatus
  pubsubname: pubsub
  metadata:

>     isRawPayload: "true"

Reviewer: Shouldn't this still be `rawPayload`?
